### DISCUSSION TITLE
Web installer does not appear

### DISCUSSION THREAD

**Discussion by adrianlshaw** [dokku/dokku#4751] [Q&A] [answered]
Blank Ubuntu 20.04 VM following the default instructions

```
 $ wget https://raw.githubusercontent.com/dokku/dokku/v0.25.1/bootstrap.sh
 $ sudo DOKKU_TAG=v0.25.1 bash bootstrap.sh
# You can then proceed to the ip address or domain name associated with your server to complete the web-based installation.
```

does not create a web installer. It's just the default nginx page. What should I do?

Here's a recording: https://asciinema.org/a/aIrCggwHAUOvhq43tsCw0Qgno

**Comment by Cellane** [comment, marked as answer]
The web installer was removed in 0.25.0 as it presented a potential security risk and essentially, was not even needed in the first place, as all the installer does can be done via CLI commands.
Removed in PR #4726, you can see the PR and the related issue for justification.

**Reply by adrianlshaw** [reply]
Thanks. It seems like the instructions in the README and the main website are out of date.

**Reply by josegonzalez** [reply]
I've removed those instructions to make it more clear.
